api/web3server.go

```diff
@@ -98,9 +103,10 @@ var (
 	errInvalidFiterID = errors.New("filter not found")
 	errInvalidBlock   = errors.New("invalid block")
 
-	pendingBlockNumber  = "pending"
-	latestBlockNumber   = "latest"
-	earliestBlockNumber = "earliest"
+	_emptyResult        *struct{} = nil // null is returned when no result was found
```

Comment:
_emptyResult is used in multiple places as nil interface{}, it makes code harder to read. For example, in function getBlockByNumber, what's an empty result? nil or &Block{}?

Reply:
good